.github/labels.json

```diff
@@ -0,0 +1,57 @@
+[
+	{ "name": "💩 template incomplete", "color": "#4E342E" },
+	{ "name": "💩 template removed", "color": "#4E342E" },
+
+	{ "name": "b¹ 🐞 code-splitting", "color": "#F44336" },
+	{ "name": "b² 🐞 scope-hoisting", "color": "#F44336" },
+	{ "name": "b³ 🐞 tree-shaking", "color": "#F44336" },
+
+	{ "name": "c¹ ⋅ discussion", "color": "#1976D2" },
+	{ "name": "c² ⋅ feedback wanted", "color": "#F9A825" },
+	{ "name": "c³ ⋅ PR welcome", "color": "#1B5E20" },
+	{ "name": "c⁴ ⋅ need more info", "color": "#6A1B9A" },
+	{ "name": "c⁵ ⋅ question", "color": "#C2185B" },
+	{ "name": "c⁶ ⋅ request for comments", "color": "#BBDEFB" },
+
+	{ "name": "p¹ ⋅ electron", "color": "#B2DFDB" },
+	{ "name": "p² ⋅ linux", "color": "#B2DFDB" },
+	{ "name": "p³ ⋅ mac", "color": "#B2DFDB" },
+	{ "name": "p⁴ ⋅ windows", "color": "#B2DFDB" },
+
+	{ "name": "pr¹ 🔧 chore", "color": "#D7CCC8" },
+	{ "name": "pr² 🔧 docs", "color": "#D7CCC8" },
+	{ "name": "pr³ 🔧 feature", "color": "#D7CCC8" },
+	{ "name": "pr⁴ 🔧 fix", "color": "#D7CCC8" },
+	{ "name": "pr⁵ 🔧 performance", "color": "#D7CCC8" },
+	{ "name": "pr⁶ 🔧 refactor", "color": "#D7CCC8" },
+	{ "name": "pr⁷ 🔧 style", "color": "#D7CCC8" },
+	{ "name": "pr⁸ 🔧 test", "color": "#D7CCC8" },
+
+	{ "name": "s¹ 🔥🔥🔥 critical", "color": "#E53935" },
+	{ "name": "s² 🔥🔥 important", "color": "#FB8C00" },
+	{ "name": "s³ 🔥 nice to have", "color": "#FDD835" },
+	{ "name": "s⁴ 💧 low", "color": "#039BE5" },
+	{ "name": "s⁵ 💧💧 inconvenient", "color": "#c0e0f7" },
+
+	{ "name": "t¹ 🐞 bug", "color": "#F44336" },
+	{ "name": "t² 📚 documentation", "color": "#FDD835" },
+	{ "name": "t³ ✨ enhancement", "color": "#03a9f4" },
+	{ "name": "t⁴ ✨ feature", "color": "#8bc34A" },
```

Comment:
Whats is the difference between an enhancement and a feature?

Reply:
"feature" is typically a new addition. "enhancement" is typically an improvement to something already existing. I can remove one or the other if it's ambiguous for Rollup

Reply:
I find it hard to draw a line between the two as any feature can be seen as an enhancement of a larger feature, so maybe we remove one to avoid thinking too much when tagging.